pages/ksphere/kommander/1.1.0-beta/release-notes/index.md

```diff
@@ -50,6 +50,33 @@ Template:
 
 -->
 
+### Version v1.1.0-rc.3 - June 11th 2020
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.15.0  |
+| **Maximum**        | 1.17.x  |
+| **Default**        | 1.17.3  |
+
+
+#### Bug Fixes
+
+- Smaller UX Bugs and Improvements
```

Comment:
A lot of component changes but they all fell under this?

Reply:
I havent seen any noteworthy changes in https://jira.d2iq.com/projects/D2IQ/versions/41236 - thats completely IMHO tho, happy to discuss/change if you want something listed. 

We should make all EM/PO/PM sign off the release notes for the stable release anyway, but I figured thats not that important for betas. 🤷

Reply:
sgtm 👍